pkg/controller/directvolumemigration/stunnel.go

```diff
@@ -70,8 +97,8 @@ data:
     pid =
     socket = l:TCP_NODELAY=1
     socket = r:TCP_NODELAY=1
-    sslVersion = TLSv1.2
     debug = 7
+    sslVersion = TLSv1.2
```

Comment:
just out of curiosity, was this the reason why previous tries of making proxy work with stunnel failing?

Reply:
@alaypatel07 nope. this was not. This actually had nothing to do with that wrong version error